lib/galaxy/config.py

```diff
@@ -697,7 +697,9 @@ def configure_logging( config ):
     # PasteScript will have already configured the logger if the
     # 'loggers' section was found in the config file, otherwise we do
     # some simple setup using the 'log_*' values from the config.
-    if not config.global_conf_parser.has_section( "loggers" ):
+    paste_configures_logging = config.global_conf_parser.has_section( "loggers" )
+    auto_configure_logging = string_as_bool( config.get( "auto_configure_logging", not paste_configures_logging ) )
```

Comment:
I think the deployer would have more control of the functionality if the documentation was updated to actually reflect what the code was doing. But I don't feel strongly enough to fight you on it and the code is easier to update, so I updated the code instead of the docs :smile:.